examples/shopping-cart/package.json

```diff
@@ -28,9 +30,14 @@
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-react": "^6.3.13",
     "babel-preset-react-hmre": "^1.1.1",
+    "cross-env": "^1.0.7",
+    "enzyme": "^2.0.0",
     "express": "^4.13.3",
     "json-loader": "^0.5.3",
+    "react-addons-test-utils": "^0.14.7",
```

Comment:
Needed if you're using React 0.14. Enzyme works with 0.13 or 0.14 but only requires `react-addons-test-utils` if you're using 0.14. Thus, they can't peerDep it, but they do document it. 

`react-dom` is required in 0.14 too, but we're using shallow rendering so we don't need it.